modules/microshift-restoring-data-backups.adoc

```diff
@@ -4,42 +4,57 @@
 
 :_content-type: PROCEDURE
 [id="microshift-restoring-data-backups-manually_{context}"]
-= Restoring {product-title} data backups manually
+= Restoring {microshift-short} data backups manually
 
-You can restore {product-title} data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When you restore a backup, you must use the entire file path.
+You can restore {microshift-short} data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Automated backups are in the `/var/lib/microshift-backups` directory by default. You can use this directory for manually backing up and restoring data by specifying it in each command. When you restore a backup, you must use the entire file path.
 
 [NOTE]
 ====
-On an `rpm-ostree` system, {product-title} backs up and restores data automatically.
+On an `rpm-ostree` system, {microshift-short} backs up and restores data automatically.
 ====
 
 .Prerequisites
+
 * Root access to the host.
 * Full path of the data backup file.
-* {product-title} is stopped.
+* {microshift-short} is stopped.
+
+[NOTE]
+====
+You can use the `microshift restore` command if the `microshift.service` status is `failed`.
```

Comment:
If we can backup and reestore when svc is `failed`, do we need this? I think it might be confusing and unexpected warning.

Reply:
Hm, wondering if we should add " ....in a healthy state" to the prereq "* {microshift-short} is stopped." Do we want users backing up and restoring from a failed state?

Reply:
I don't know. Since we're not imposing any limits I would just not mention it.

I can think of scenario where someone forgot to back up the data and upgraded microshift, but microshift fails (e.g. version checking fails - so there are no changes to the data) - and they want to create backup at that point, before downgrading. Technically the data is valid and healthy, just svc refused to run.

After recent changes to automated backup and restore, I'd let them do what they need to do (perhaps someone would create a backup of failed service to restore it on different device and investigate?)

BTW. `microshift backup` will not overwrite existing backup (the path must not exists), so we don't have to worry about user losing "healthy" backup

@dhellmann WDYT?

Reply:
I agree with removing the caveats.

The current text looks OK to me. I'll defer to @pmtk for final SME approval.